examples/showcase/tsl/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>TSL • A-Frame</title>
+    <meta name="description" content="TSL • A-Frame" />
```

Comment:
Would probably name this example 'webgpu' instead of tsl?

Reply:
The main reason I did this example is to play with TSL that is currently only available with WebGPURenderer, but that example may also run with WebGL2 for systems that doesn't support WebGPU, so I would like to keep TSL here in the title.
I renamed directory to webgpu and changed the title to "WebGPURenderer and TSL" so that matches what I put in the root index.html